apps/judicial-system/backend/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -235,4 +235,12 @@ export class Defendant extends Model {
   @Column({ type: DataType.STRING, allowNull: true })
   @ApiPropertyOptional({ type: String })
   alternativeServiceDescription?: string
+
+  @Column({ type: DataType.STRING, allowNull: true })
+  @ApiPropertyOptional({ type: String })
+  verdictForDefendant?: string
+
+  @Column({ type: DataType.ARRAY(DataType.STRING), allowNull: false })
+  @ApiProperty({ type: String, isArray: true })
+  informationForDefendant!: string[]
```

Comment:
Please use an enum here. You can find an example of how that is done above. `VerdictAppealDecision` is a good example. Note there is a sligth difference becuase this is an array. A good example of how enum arrays are handled is the `legalProvisions` field in the case model.